### ISSUE TITLE
Email: Comments on WCAG 3.0 draft (7- Complicated Conformance & Scoring)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#486]
[Comment from Email](https://lists.w3.org/Archives/Public/public-silver/2021Mar/0002.html): 
from [Word docx](https://lists.w3.org/Archives/Public/public-silver/2021Mar/att-0002/Review_Comments_for_WCAG_3.0_-_Vanderheiden.docx) 

3)	Complicated conformance and scoring methods.  
It is fine to have a complicated scoring or rating methods for a guidance document (e.g. one that is not called WCAG and which is not intended to be adopted in regulation).  However, if one wants to create a document that would be used in the legal and regulatory arena, then one has to stick with objective measures.  
It should be noted that even when creating WCAG 2.0, where guidelines were first sorted into 3 levels (A, AA, and AAA), it was always understood that we were simply giving regulatory bodies a choice of choosing either the provisions on Level A or the provisions in Levels A and AA (as a group), as the provisions for their standard. There was never any belief they were going to have two levels of conformance in their standard.  We simply gave them two choices as to where they wanted to draw the line for accepting provisions into their standard.
It should also be also noted that we also created a level AAA that contain provisions that we tried but just could not get into level AA because they did not meet the criteria.   We stretched the requirements here in creating Level AAA because wanted those concepts and needs to be in the same document  -- so that they would be able to get the attention of everyone -- and so that we could write techniques and strategies about them.  We also added lots of techniques and strategies that went beyond what was required in level and AA and attached them as additional guidance for the same reason.  I see a similar strategy being tried in WCAG 3 – but not identified so clearly.  In WCAG 2 we clearly stated that Level AAA was different than A and AA and that it should not be required across all content because they did not meet the tests for what could be required for a provision that would be objective and/or that could apply across all web content. 
I mention these here not in defense or justification of WCAG 2.0, which needs to stand on its own and be critiqued and improved as was done in 2.1. and 2.2. I mention these here so that there is a better understanding of the frustrations and the barriers to be faced in the WCAG 3 work – and which, as I highlight here, were also faced before, and how we went about trying to deal with them.  Some “missing” issues were not covered in earlier versions not because those WCAG teams didn't care, and not because there were not people in those working groups who were interested in and worked hard for all of these disability groups and issues,  -- but simply because they/we could not find ways to cover these issues… could not find language for provisions that would work.”    
"But we aren't writing a regulation – so why do we have to follow those rules/constraints?"
Yes, this is true.  This is an accessibility standard  - not a regulation.   We said the same thing when we were writing WCAG 2.0.  But we quickly realized that if we wanted those who were writing regulations to use our document, that we need to conform to their rules.  If we didn’t/ don't follow their rules, then they will not be able to use our document directly.  They will have to make up their own provisions or pick and choose among ours to find those that do meet the rules that they need to follow, and leave all the rest behind.  
Again, it is suggested that a decision be made as to whether (A) this document meant to be a really great global holistic document on accessibility, or whether (B) it's meant to be the next chapter in the WCAG line of accessibility standards that were/are designed to be able to be used in legal and regulatory spheres.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
